### PR TITLE
Bug 1206194 - Imagecompare: Enable wifi hotspot to capture RTL text

### PR DESCRIPTION
**Pull request by nojunpark**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=32925c3fcb67d128404abf70df3c7ebc6f45f3da

File: tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/internet_sharing.py [hunk 1]
```diff
@@ -48,3 +58,4 @@ def confirm_security_settings(self):
             Wait(self.marionette).until(
                 expected.element_displayed(*self._security_selector_locator))
 
+
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/internet_sharing.py]: Nit, return too many

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=1ca66b48ac7a75258cb0709228282bb9bd6f8ad1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=307e1b8fbb9bfdae8ca28839986a2303220e017f

**Pull request merged**